integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/DDLStrategy.scala

```diff
@@ -224,15 +224,32 @@ class DDLStrategy(sparkSession: SparkSession) extends SparkStrategy {
           && (tableDesc.provider.get.equals("org.apache.spark.sql.CarbonSource")
           || tableDesc.provider.get.equalsIgnoreCase("carbondata")) =>
         val updatedCatalog =
-          CarbonSource.updateCatalogTableWithCarbonSchema(tableDesc, sparkSession)
+          CarbonSource.updateCatalogTableWithCarbonSchema(tableDesc, sparkSession, None)
```

Comment:
take the default value as None in method updateCatalogTableWithCarbonSchema instead of passing from  here

Reply:
okay